sbi/analysis/plot.py

```diff
@@ -1569,6 +1569,9 @@ def _sbc_rank_plot(
         plot_type in plot_types
     ), "plot type {plot_type} not implemented, use one in {plot_types}."
 
+    if legend_kwargs is None:
```

Comment:
In this way, the legend is always drawn. I think, it is OK to keep it like this as a user of this function can always turn it off using the `fig, ax` objects. Just wanted to raise this here as it enforces the legend.

Reply:
Good point, I was also thinking about this and whether we should add a flag for the legend. But it was like this before and I agree, that we could just leave it to the user to remove or change the legend using the `fig, ax` objects.

Reply:
Sounds good.